.buildkite/test-pipeline.yaml

```
@@ -591,8 +624,17 @@ steps:
   commands:
     - pytest -v -s models/test_transformers.py \
                    models/test_registry.py \
-                   models/test_utils.py \
-                   models/test_vision.py
+
+
+- label: Basic Models Test (CPU) # 5min
+  timeout_in_minutes: 10
+  torch_nightly: true
+  source_file_dependencies:
+  - tests/models
+  no_gpu: true
+  commands:
+    - pytest -v -s models/test_utils.py
+    - pytest -v -s models/test_vision.py
```

Should vision and utils be removed from the GPU version of this job?

```suggestion
    - pytest -v -s -m 'cpu_test' models
```